jury-advice.md: Add that sample outputs will be visible in DOMjudge (#22)
WISVCH/nwerc-2024: content/jury-advice.md

  Attempts to start extra processes or threads may
  lead to a judgement of "Run Time Error".

- After receiving the judgement of your submission,
  you are able to view the output of your submission and the corresponding verdict

>   for each of the samples from the DOMjudge interface.

Reviewer: ```suggestion
  of each of the samples in the DOMjudge interface.
```
